tests/test_dictionary.py

```diff
@@ -131,6 +131,23 @@ def test_merge_dictionaries_image():
     assert merged_dict.image == moco_dict.image
 
 
+@pytest.mark.no_connection
+def test_merge_dictionaries_new_instance():
+    coco_dict_path = f"{PATH_RESOURCE}comkit/com-kit.xdf"
+    moco_dict_path = f"{PATH_RESOURCE}comkit/core.xdf"
+    dict_a = EthernetDictionaryV2(coco_dict_path)
+    dict_b = EthernetDictionaryV2(moco_dict_path)
+    dict_c = dict_a + dict_b
+
+    assert id(dict_c) != id(dict_a)
+    assert id(dict_c) != id(dict_b)
+
+    dict_d = dict_b + dict_a
+
+    assert id(dict_d) != id(dict_a)
```

Comment:
I'm not very familiar with how deepcopy works. Could we check that is has created clones of the objects we want, but not others, with the same id checking you're doing.
For example a register must be copied, An enum instance (let's say a datatype) should not...

Reply:
Done